packages/synapse-react-client/src/components/entity/page/title_bar/TitleBarProperties.integration.test.tsx

```diff
@@ -226,7 +258,7 @@ describe('TitleBarProperties', () => {
             endpointUrl,
             bucket,
             fileKey,
-          },
```

Comment:
The remaining changes in this file are related to linting errors found after colocating test file with component